integration-tests/hibernate-reactive-mssql/pom.xml

```diff
@@ -178,87 +180,6 @@
                 <quarkus.native.add-all-charsets>true</quarkus.native.add-all-charsets>
             </properties>
         </profile>
-
-        <profile>
-            <id>docker-mssql</id>
-            <activation>
-                <property>
-                    <name>start-containers</name>
-                </property>
-            </activation>
-            <properties>
-                <!-- Careful: SQL Server refuses to work with trivial passwords. -->
-                <mssqldb.sa-password>yourStrong(!)Password</mssqldb.sa-password>
-            </properties>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>io.fabric8</groupId>
-                        <artifactId>docker-maven-plugin</artifactId>
-                        <configuration>
-                            <images>
-                                <image>
-                                    <name>${mssql.image}</name>
-                                    <alias>quarkus-test-mssqldb</alias>
-                                    <run>
-                                        <ports>
-                                            <port>1435:1433</port>
-                                        </ports>
-                                        <env>
-                                            <ACCEPT_EULA>Y</ACCEPT_EULA>
-                                            <MSSQL_SA_PASSWORD>yourStrong(!)Password</MSSQL_SA_PASSWORD>
-                                        </env>
-                                        <log>
-                                            <prefix>MS SQL Server:</prefix>
-                                            <date>default</date>
-                                            <color>cyan</color>
-                                        </log>
-                                        <wait>
-                                            <log>SQL Server is now ready for client connections</log>
-                                            <!-- Unfortunately booting this is slow, needs to set a generous timeout: -->
-                                            <time>40000</time>
```

Comment:
FWIW I checked and the default timeout for MSSQL in test containers is... 240 seconds. So we should be good, though if things start failing CI will get quite slow.